scripts/travis/compile.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# TODO: Would probably be faster to use -j2 NCPUS=1 as for other steps
+# but many dependency compilations seem not parallel-safe
+NCPUS=1
+
+for x in base/logger base/remote modules/emulator base/utils base/db base/settings base/visualization base/qaqc modules/data.atmosphere modules/data.land modules/priors modules/uncertainty base/workflow modules/allometry modules/meta.analysis modules/assim.batch modules/assim.sequential modules/benchmark modules/data.hydrology modules/data.remote modules/photosynthesis models/template models/ed modules/rtm models/biocro models/clm45 models/dalec models/dvmdostem models/fates models/gday models/jules models/linkages models/lpjguess models/maat models/maespa models/preles models/sipnet base/all ; do
+    key=$(echo "make/$x" | sed 's#/#_#g')
```

Comment:
This order looks like it's derived from current dependency relationships. Will it need to change when package dependencies change?